src/turbo-stream.ts

```diff
@@ -238,22 +271,24 @@ export function encode(
                     if (Array.isArray(id)) {
                       controller.enqueue(
                         textEncoder.encode(
-                          `${TYPE_ERROR}${deferredId}:[["${TYPE_PREVIOUS_RESOLVED}",${id[0]}]]\n`
+                          `${TYPE_PROMISE}${TYPE_ERROR}${deferredId}:[["${TYPE_PREVIOUS_RESOLVED}",${id[0]}]]\n`
```

Comment:
I did change the existing transport format a bit here - this is not strictly necessary, but this way, we have less of an explosion of types & type cases, and a bit more consistency.

It went from
`E5` to `PE5` to signal that promise 5 errored.

Further down I introduced 
`TE5` to signal that stream 5 errored.

This "combination of characters" is kinda necessary as otherwise I would have to introduce another new signal for "stream done" as well as "stream errored" and re-use a few characters in ways that do not necessarily make sense

So instead, further down I introduce

* `T5` for "new value in stream 5"
* `TO5` for "stream 5 finished"
* `TE5` for "stream 5 errored"